lib/widgets/app.dart

```diff
@@ -12,6 +12,7 @@ class ZulipApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final theme = ThemeData(
+      fontFamilyFallback: const <String>['sans-serif', 'Noto Color Emoji'],
```

Comment:
Can you say more about what this line is doing and why?

The commit message says "correctly propogate emoji font fallback", but this doesn't appear to be propagating anything; instead it's introducing something new.

Reply:
ack, added a comment explaining the reason and moved to a separate commit.